docs/whats-new.md

```diff
@@ -67,3 +67,10 @@ Alfresco SDK 4.1 provides support for Alfresco 6.2.x.
 ## Support for Alfresco 7.0.x, 7.1.x, and 7.2.x
 
 Alfresco SDK 4.4 provides support for Alfresco 7.0.x, 7.1.x, and 7.2.x.
+
+## Support for Java 17
+[Java 17](https://openjdk.java.net/projects/jdk/17/) is the latest stable long term (8 year support) version after Java 11. Alfresco 7.3.x already offers
+support for this version of the Java platform.
+
+Alfresco SDK 4.5 has been modified to add support for Java 17 as well. This way, if you're working as a developer in customisations for Alfresco 7.3.x you must
```

Comment:
We still compile sdk in java 11 though, so java 17 "can" be used by someone else when adding their extensions, not "must"